index.html

```diff
@@ -32,6 +50,22 @@ <h1 class="heading">Add a new book</h1>
       </div>
     </form>
 
+    <div class="contactsection">
+        <h1 class="heading">Contact Information</h1>
+        <p>Do you have any questions or you just want to say "Hello"?<br> You can reach out to us</p>
+        <ul class="contactul">
+          <li>Email: sja.affu765@gmail.com</li>
+          <li>Phone Number: 00966572705741</li>
+          <li>Address: Jeddah ,KSA</li>
+        </ul>
+    </div>
```

Comment:
- [ ] OPTIONAL - Consider adding margin and padding to elements, it's important elements have spacing between them. Most contents on the page are pretty crowded together.